src/main/java/uk/gov/pay/directdebit/common/exception/JsonMappingExceptionMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.pay.directdebit.common.exception;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Priority;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Priority(1)
```

Comment:
this is good to know, and something we should probably stick in all our apps - I have had difficulties debugging stuff on a number of occasions because the default `JsonProcessingException` handling appears totally silent